### PR TITLE
feat(dashboard): revamp the search input #3379

### PR DESCRIPTION
**Pull request by nisha-laur**
Fixes #
This pull request addresses the "Revamp: Search Input in Dashboard #3379" issue in CircuitVerse. 

#### Describe the changes you have made in this PR -
It aims to improve the search input appearance in the dashboard making it look more appealing by modifying some css code and adding the class 'form-control' and  'form-input' .  

### Screenshots of the changes (If any) -
A) When Input tag is not clicked and selected 
Before :
image

After:
image

B)A) When Input tag is  clicked and selected 
Before: It shows plain black color which clearly does not matches with the website theme .
image
After:
image

### CODE REVIEW

**Comment by tanmoysrt** [comment]
Hi @nisha-laur 
Thanks for raising the PR.

As per the code of conduct, you must get assigned to the issue before raising the PR.
As you are first-time contributor and none is assigned to that issue, comment there #3379, and I will assign the issue to you.

From the next contributions, follow the flow.

**Comment by nisha-laur** [comment]
Thank you, I will do that.

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -323,15 +323,16 @@
 
   input {
     border: 0;
-    border-left: 1px solid $form-border-grey;
+    // border-left: 1px solid $form-border-grey;
```
Comment (tanmoysrt) [inline app/assets/stylesheets/users.scss:326]:
```suggestion
```

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -323,15 +323,15 @@
 
   input {
     border: 0;
-    border-left: 1px solid $form-border-grey;
+    border: 1px solid #42b983;
```
Comment (tanmoysrt) [inline app/assets/stylesheets/users.scss:326]: Use the defined variables, don't use direct hex code for color

File: app/assets/stylesheets/users.scss [hunk 3]
```diff
@@ -323,15 +323,15 @@
...
     border-radius: 0;
     font-size: 16px;
     margin: 0;
     padding: 7px 10px 7px 35px;
     transition: .2s ease-in-out;
 
     &:focus {
-      border-color: $primary-green;
+      border-color: green;
```
Comment (tanmoysrt) [inline app/assets/stylesheets/users.scss:334]: Use the defined variables, don't use direct hex code for color

File: app/assets/stylesheets/users.scss [hunk 4]
```diff
@@ -350,12 +350,14 @@
 }
 
 .users-circuits-search {
+
```
Comment (tanmoysrt) [inline app/assets/stylesheets/users.scss:353]: remove blank space

File: app/assets/stylesheets/users.scss [hunk 5]
```diff
@@ -350,12 +350,14 @@
...
 
 .users-circuits-search {
+  
   align-items: center;
   display: flex;
   flex-wrap: nowrap;
   justify-content: center;
   padding-top: 20px;
   width: 100%;
+
```
Comment (tanmoysrt) [inline app/assets/stylesheets/users.scss:360]: remove blank space

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

**Review by tanmoysrt — APPROVED** [review]
Looks Good !

**Pull request merged**